docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -0,0 +1,91 @@
+# Required Software
+
+1. [Visual Studio 2015 with Update 1](http://go.microsoft.com/fwlink/?LinkId=691129). _You need Update 1_.
+2. Visual Studio 2015 Extensibility Tools. If you already installed Visual Studio, choose "Modify" from the Programs and Features control panel, and check "Visual Studio Extensibility".
+
+# Getting the Code
+
+1. Clone https://github.com/dotnet/roslyn
```

Comment:
Perhaps? I'm not sure I want to include "how to use GitHub" in our instructions, versus assuming that as a base.